sound/soc/soc-pcm.c

```diff
@@ -745,6 +749,9 @@ static int soc_pcm_close(struct snd_pcm_substream *substream)
 	for_each_rtdcom(rtd, rtdcom) {
 		component = rtdcom->component;
 
+		if(component->driver->ignore_module_refcount)
```

Comment:
if (component)

Reply:
maybe we should rename ignore_module_refcount as check_module_refcount_on_open or something more meaningful than 'ignore' and then use it.

Reply:
@plbossart sorry about the style fixes. I'll fix them. 
I agree it makes sense to rename this field. I havent been able to come up with a nice name that indicates what we're really trying to do here. Let me think about it and rename it today.

Reply:
done, I've rename it to module_get_upon_open. hopefully thats a bit more indicative of what we want to achieve.